docker/scripts/common.sh

```diff
@@ -88,18 +88,17 @@ e2e_test_op_proof(){
     local root=$(get_root_dir)
     local l1_rpc=$1
     local l2_rpc=$2
-    local network_config=$3
     local privkey=$4
     local txhash=$5
     local relayer=${root}/relayer/relayer-node18-linux-x64
 
     echo Starting OP relayer message prover...
-    ${relayer} prove-op-message \
-                --l1-rpc-url $l1_rpc \
-                --l2-rpc-url $l2_rpc \
-                --custom-network $network_config \
-                --wallet-private-key $privkey \
-                --l2-transaction-hash $txhash
+    timeout 5m bash -c "until ${relayer} prove-op-message \
+                                         --l1-rpc-url '$l1_rpc' \
+                                         --l2-rpc-url '$l2_rpc' \
+                                         --wallet-private-key '$privkey' \
+                                         --l2-transaction-hash '$txhash'; \
+    do sleep 1s; done"
 }
```

Comment:
I think this entire function should not be in `common.sh` but rather in `optimism.sh`, because it is Optimism-specific.

Reply:
moved